orca-queue/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Sorry, I'm not familliar with this. Why do we need it?

Reply:
this give us the ability to mock or spy final classes, in this case, I needed to `spy(NoopRegistry)` in the tests [here](https://github.com/spinnaker/orca/pull/4255/files#diff-5030240d38e2011d634b6cf4ab9ef2868655238b0cb1f254c30ba0e4e3f03618R108)

Reply:
Thanks!